src/components/ActionBar/index.js

```diff
@@ -2,3 +2,4 @@ export { default as MActionBar } from './src/ActionBar.vue';
 export { default as MActionBarLayer } from './src/ActionBarLayer.vue';
 export { default as MActionBarButton } from './src/ActionBarButton.vue';
 export { default as MResponsiveActionBar } from './src/ResponsiveActionBar.vue';
+export { default as MInlineActionBar } from './src/InlineActionBar.vue';
```

Comment:
I'm kind of confused about the differences between ResponsiveActionBar and InlineActionBar or why you would use one over the other. I can see that InlineActionBar has more functionality, but is there a scenario to simplify both into a single MActionBar?

Reply:
After the refactor the only place `ResponsiveActionBar` is used is from local doc dev so I can test that both `ActionBar` and `InlineActionBar` work how I expect, lol. I'll remove the `ResponsiveActionBar` export and remove it from the docs, and try to add more explanation on when to use `ActionBar` vs `InlineActionBar`, although if it's being used in any kind of responsive scenario the answer almost always will be `InlineActionBar`. Merging the 2 would be a breaking change so I'd like to hold off on that before we get some feedback from users, but yes, the end goal would eventually be to have a single `ActionBar` component.

Reply:
I removed `ResponsiveActionBar` and updated the `ActionBar` docs.